src/Datadog.Trace.AspNet/TracingHttpModule.cs

```diff
@@ -72,6 +74,13 @@ private void OnBeginRequest(object sender, EventArgs eventArgs)
                     return;
                 }
 
+                // Only one HttpModule will respond to ASP.NET lifecycle events
+                // Do nothing if another HttpModule has already been run
+                if (httpContext.Items[_httpContextOwningModuleKey] is TracingHttpModule)
```

Comment:
I assume this accounts for it not being in the Items list?

Reply:
That's correct. If the key is not present, then this will return null, and a null value will cause the `is` check to return `false`. I thought to follow the previous convention with this style of check, but would there be another way to express this concisely?